litr-muxers/src/main/cpp/build_ffmpeg.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+FFMPEG_MODULE_PATH=$1
+NDK_PATH=$2
+HOST_PLATFORM=$3
+ENABLED_DECODERS=("${@:4}")
+JOBS=$(nproc 2> /dev/null || sysctl -n hw.ncpu 2> /dev/null || echo 4)
+echo "Using $JOBS jobs for make"
+COMMON_OPTIONS="
```

Comment:
This script is based upon ExoPlayer's ffmpeg module, except with *everything* disabled. I've only enabled what is required to match the capabilities of Android's MediaMuxer, with the addition of segmented support.